content/vsphere-cpi.md

```diff
@@ -65,6 +65,7 @@ Schema for `cloud_properties` section:
 * **disk** [Integer, required]: Ephemeral disk size in megabytes. Example: `10240`.
 * **cpu\_hot\_add\_enabled** [Boolean, optional]: Allows operator to add additional CPU resources while the VM is on. Default: `false`. Available in v21+.
 * **memory\_hot\_add\_enabled** [Boolean, optional]: Allows operator to add additional memory resources while the VM is on. Default: `false`. Available in v21+.
+* **upgrade\_hw\_version** [Boolean, optional]: Upgrades the virtual hardware version of a virtual machine to the latest supported version on the ESXi host. This property can also be specified in global configuration which will be overridden by VM Type property. By default this property is false.
```

Comment:
Upgrades virtual machines to latest virtual hardware version supported on the ESXi host. Overrides the global upgrade\_hw\_version.  Available in v49+ ?